cmd/cli/cmd/deprovision/kind.go

```diff
@@ -0,0 +1,22 @@
+package deprovision
+
+import (
+	"capact.io/capact/internal/cli/deprovision"
+	"github.com/spf13/cobra"
+)
+
+func NewKind() *cobra.Command {
+	var opts deprovision.KindOptions
+
+	cmd := &cobra.Command{
+		Use:   "kind",
+		Short: "Delete local kind cluster",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return deprovision.Kind(cmd.Context(), opts)
+		},
+	}
+	cmd.Flags().StringVar(&opts.Name, "name", "kind-dev-capact", "cluster name, overrides KIND_CLUSTER_NAME, config (default kind-dev-capact)")
```

Comment:
`default` message is already generated by cobra.
```suggestion
	cmd.Flags().StringVar(&opts.Name, "name", "kind-dev-capact", "cluster name, overrides KIND_CLUSTER_NAME, config")
```

Reply:
setting env `KIND_CLUSTER_NAME` doesn't work for me

Reply:
copy paste from Kind. Do we need to support env variables?

Reply:
I don't thinks so :) nice but not required